_overviews/scala3-book/scala-for-python-devs.md

```diff
@@ -186,7 +186,7 @@ These examples demonstrate how to create variables in Python and Scala.
     </tr>
     <tr>
       <td class="scala-block">
-        <code>val movies = Map(
+        <code>val x = Map(
```

Comment:
Sorry to be picky, but `movies` was a better name than `x`, IMHO. If we change something here, we should rather change the Python code to use `movies` as well.

Reply:
I thought that too, but  `x` was used throughout the documentation so I felt it looked weird.